src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import 'product-card';
+@import 'stars';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
```

Comment:
The font-family declaration should list 'Arial' before 'sans-serif' to ensure the fallback to a generic sans-serif font if 'Roboto' is not available. This is a minor issue related to the code style rules.